XenAdmin/Wizards/RollingUpgradeWizard/RollingUpgradeWizardPrecheckPage.cs

```diff
@@ -234,6 +226,13 @@ protected override List<CheckGroup> GenerateChecks(Pool_patch patch)
                     automatedUpdateLicenseChecks));
             }
 
+            //Check for update file availability
+            if (ApplyUpdatesToNewVersion)

```

Comment:
Could we squash this if block with the previous if block (line 219), as they are the same condition? And maybe could first check if c-f-u downloadable and then check license status, based on normal logic order.

Reply:
Hmmm, I was wondering about the order, but in the end I thought that the license check may be more important: if, for example, the cfu check fails due to a connectivity error, is there a point in the user taking time to fix it since they will still be unable to install automated updates on their unlicensed pool? @MihaelaStoica could you offer a third opinion on this?

Reply:
We should do the license check first, as that's what we usually do in other places: if a feature is not available due to the license status, then no other conditions are checked. 

I am thinking that maybe we should combine this checks into one, which will be the "automated updates availability" check, cheking the license +  connectivity. In most cases these checks will succeed, especially the license one (because the RPU is a licensed feature and therefore we won't reach the prechecks page with an unlicensed pool). So having two groups for these checks seems a bit too much.